README.md

```diff
@@ -74,7 +74,7 @@ Translations
 ------------
 
 Changes to translations as well as new translations can be submitted to
-[Bitcoin Core's Transifex page](https://www.transifex.com/projects/p/bitcoin/).
+[Bitcoin Classic's Transifex page](https://www.transifex.com/projects/p/bitcoin/).
```

Comment:
The page https://www.transifex.com/bitcoinclassic/bitcoinclassic/ exists.  Is this under control by the Classic team?